src/components/MovieHeadInfo/MovieHeadInfo.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Stars } from '../common/Stars'
+import { Raiting } from '../common/Raiting'
+import styles from './MovieHeadInfo.scss'
+
+const timeFormat = (time) => {
```

Comment:
hm, actually it’s not related to the MovieHeadInfo
let's move it outside to the utils folder
and I think that the helper should return an object or something with correct time format